src/Transports/MassTransit.ActiveMqTransport/Configuration/Configurators/ConfigurationHostSettings.cs

```diff
@@ -53,10 +56,50 @@ public ConfigurationHostSettings(Uri address)
         public Uri HostAddress => _hostAddress.Value;
         public Uri BrokerAddress => _brokerAddress.Value;
 
+        static readonly LogMessage<string> _logDebug = LogContext.Define<string>(LogLevel.Debug, "Connection info: {info}");
+
         public IConnection CreateConnection()
         {
             var factory = new NMSConnectionFactory(BrokerAddress);
-            return factory.CreateConnection(Username, Password);
+
+            if (ActiveMqArtemisSupport.EnableExtraConnectionLogging)
+            {
+                _logDebug($"Creating connection to {BrokerAddress}");

```

Comment:
The main point of using `LogMessage` is to specify correct number of arguments, and format - so your message will be built when `logger.IsEnabled(level)`. But your are using string concatenation again.

Long story short, your `LogMessage` should already define message format and you just pass an arguments, do not build new string!